openedx/core/djangoapps/auth_exchange/views.py

```diff
@@ -145,7 +146,14 @@ def post(self, request):
         if not hasattr(request.user, 'backend'):
             request.user.backend = self._get_path_of_arbitrary_backend_for_user(request.user)
 
-        if not self._is_grant_password(request.auth):
+        access_token = request.auth
```

Comment:
Maybe this could move to an `else` case below, with a comment about it being a bearer token?

Reply:
Sure.